intermediate_data_model.py

```diff
@@ -0,0 +1,436 @@
+import re
+
+import utils
+from purchase_invoice import SupplierItem, PurchaseInvoice
+from settings import NKK_ACCOUNTS, KORNKRAFT_ACCOUNTS
+
+
+class InvoiceIntermediateModel:
```

Comment:
the class should be named `PurchaseInvoiceData`, and the file should be named `purchase_invoice_data.py`.

Reply:
aha, I think the file should be better called `purchase_invoice_parser.py`, because the vast majority of the code lines is about parsing.